migrate.js

```diff
@@ -0,0 +1,38 @@
+const taskcluster = require('taskcluster-client');
+const auth = new taskcluster.Auth();
+
+const migrate = async () => {
+  try {
+    const clients = await auth.listClients();
+
+    // Migrate mozillians-unvouched
+    await auth.createRole('everybody', {
+      scopes: await auth.role('mozillians-unvouched'),
+      description: 'Role assigned to everybody. It should only have the scopes required to run the tutorial, and nothing that might harm other users of Taskcluster.',
+    });
+
+    clients
```

Comment:
I think we want to use the list of roles that start with `mozilla-user:` here, rather than the list of clients.  The change to logins won't affect clients (aka permacreds).